src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED("red"),
+    GREEN("green"),
+    BLUE("blue"),
+    BLACK("black"),
+    WHITE("white"),
+    YELLOW("yellow"),
+    BROWN("brown"),
+    GREY("grey");
+
+    private final String properOutput;
+    Colors(String properOutput) {
+        this.properOutput = properOutput;
+    }
+
+    public String getProperOutput() {
+        return properOutput;
+    }
+}
```

Comment:
Good point. But you either can use `Color.name().toLowerCase()`, see previous comment.